### PR TITLE
fix: navbar disappears on scrolling through out the page

### PR DESCRIPTION
**Pull request by itzabhinavarya**
## What does this PR do?

This PR will fix the issue when we scroll in Y direction through out the page the navbar is disappears, I have fixed this
image

Fixes # (issue)

image

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand bits
- [x] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

File: apps/formbricks-com/components/shared/Header.tsx
Comment (itzabhinavarya) [inline apps/formbricks-com/components/shared/Header.tsx:1]: @ShubhamPalriwala @jobenjada Sir, Only this file i have changes for the PR no. #1106 . Please check/compare,if everthing looks good then merge only this file.

**Comment by itzabhinavarya** [comment]
@ShubhamPalriwala  @jobenjada  Sir, Please have a look on this PR. Feature reference no. #1054 . Which i have solved recently.

**Comment by ShubhamPalriwala** [comment]
Hey, for this feature, this is how we would like to proceed further:
https://github.com/formbricks/formbricks/pull/1076#issuecomment-1757537001

And we'd be giving the person who created and was assigned the issue another 24 hours to push their work and if thats not done, then we'll open it up and maybe you can open a PR then about working on what we actually want.

Hence, closing this for now!

**Comment by itzabhinavarya** [comment]
Ok Sir. BTW is my solution correct?

**Pull request closed without merging**